Added new Series F fundraising announcement blog post (#46641)
supabase/supabase: apps/www/_blog/2026-06-04-supabase-series-f.mdx

---
title: 'Supabase Series F'
description: 'Supabase has raised a $500M Series F at a $10B pre-money valuation, led by GIC.'
author: paul_copplestone
imgSocial: series-f/supabase-series-f.png
imgThumb: series-f/supabase-series-f.png
categories:
  - company
tags:
  - supabase
date: '2026-06-04T12:00:00'
toc_depth: 2
---

Supabase has raised a $500M Series F at a $10B pre-money valuation. It’s led by GIC. All existing investors joined the round, Stripe made a second investment, and Salesforce Ventures joined as a new investor.

The round is for three things:

- Accelerate development of open source / Postgres tools.
- To support our growth.
- Liquidity for the employees.

## Open source development

Today we're releasing Multigres v0.1 alpha. It’s open source and self-hostable.

Postgres is hard to run at scale. Multigres is a scalable operating system for Postgres that provides high availability and operational simplicity. In a future release it will provide Vitess-grade horizontal scaling.

> The alpha status is to signal that it’s ready to try, but not yet production-ready. We’ll get there in a few months. If you’re running a large Postgres workload you can apply to be a Multigres Partner to help shape the direction of Multigres.

Reviewer: ```suggestion
The alpha status is to signal that it’s ready to try, but not yet production-ready. We’ll get there in a few months. If you’re running a large Postgres workload you can apply to be a Multigres Partner and help shape the direction of Multigres.
```

need to add link

> Multigres will be available on the platform when it moves out of alpha. Together with our work on OrioleDB, which we aim to make production-ready this year, we believe that we will have a very complete solution - solving many hairy Postgres problems including bloat, connection pooling, and Jepsen-level high availability.

Reviewer: ```suggestion
Multigres will be available on the platform when it moves out of alpha. Together with our work on OrioleDB (targeting production-readiness this year) we believe that we will have a complete Postgres solution that solves many hairy Postgres problems including bloat, connection pooling, Jepsen-level high availability. Sharding is also on the menu - we'll share more details later.
```

## Supabase growth

> In the eight months since our last fundraising event, our user base has more than doubled, the number of databases launched grew 600% year over year. Nearly 10 million developers now build on Supabase. This year we’ve been accelerating as Claude Code and Codex are turning everyone into builders. Over 60% of databases are launched by some sort of AI tool.

Reviewer: ```suggestion
In the past year, database launches on Supabase have grown 600%. More than 60% of new databases are launched by some sort of AI tool. 

Nearly 10 million developers build on Supabase today, more than doubling since our last fundraising announcement eight months ago. Growth is accelerating since January as Claude Code and Codex expand the number of people who can build. 
```

![Cumulative Supabase users from 2020 to 2026, growing to nearly 10 million](/images/blog/series-f/supabase-users-growth.png)

> A lot of our growth is also thanks to our partners, platforms that build on top of Supabase. Supabase for Platforms has been the fastest growing product over the last year.

Reviewer: ```suggestion
A lot of our growth is also thanks to our partners, platforms that build on top of Supabase. [Supabase for Platforms](/docs/guides/integrations/supabase-for-platforms) has been the fastest growing product over the last year.
```

> Acceleration at our current scale brings a unique set of challenges. We’re using this funding to invest in the not-so-visible platform features: performance, reliability, and customer support teams to help out. We’re investing in tooling that can help our customers to operate their databases autonomously. Postgres can be hard to manage, even for experienced DBAs, and we have a lot of ideas on how to solve that.

Reviewer: ```suggestion
Managing infrastructure at our scale brings a unique set of challenges. We’re using this funding to invest in the not-so-visible platform features: performance, reliability, and support teams to help our customers. Postgres can be hard to manage, even for experienced DBAs, and we're investing in tooling that helps builders run their databases autonomously.
```

## Employee secondaries

> Supabase has employees in over 50 countries now. We are always looking for the “best person for the job”, no matter where they live. We don’t do geo-adjusted salaries - you get paid for the result of your work, not the city you live in.

Reviewer: ```suggestion
Supabase has employees in over 50 countries now. We are always looking for the “best person for the job”, no matter where they live. We don’t do geo-adjusted salaries - people get paid for the result of their work, not the city they live in.
```
